src/crystal/system/unix/urandom.cr

```diff
@@ -1,5 +1,4 @@
-# TODO: replace with `flag?(:unix) && !flag?(:openbsd) && !flag?(:linux)` after crystal > 0.22.0 is released
-{% skip_file() if flag?(:openbsd) && flag?(:linux) %}
+{% skip_file if flag?(:unix) && !flag?(:openbsd) && !flag?(:linux) %}
```

Comment:
I wonder why this checks to only skip on `unix` that isn't openbsd or linux if this file would only be `require`d for unix already? This is kind of telling "dont skip if it is windows", which doesn't make sense.

Reply:
Yep, there's sth off here, it erroneously skips file for OS X.